src/relational.cpp

```diff
@@ -162,6 +162,28 @@ using namespace cpp11;
 	return make_external_prot<RelationWrapper>("duckdb_relation", prot, res);
 }
 
+[[cpp11::register]] SEXP rapi_rel_filter2(data_frame df, duckdb::conn_eptr_t con, list exprs) {
```

Comment:
Can we move these to a separate `.cpp` file and keep them in the same order as here?

Reply:
This is still valid. Can you please follow up?